clisops/ops/average.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+from typing import List, Optional, Tuple, Union
+
+import xarray as xr
+from roocs_utils.parameter.dimension_parameter import DimensionParameter
```

Comment:
I've been trying to run the tests but the `roocs-utils` I have (even with master) doesn't seem to have this module, so tests are failing at the collection step.